source/common/http/conn_manager_utility.cc

```diff
@@ -176,14 +176,49 @@ Network::Address::InstanceConstSharedPtr ConnectionManagerUtility::mutateRequest
     request_headers.insertRequestId().value(uuid);
   }
 
-  if (config.tracingConfig()) {
-    Tracing::HttpTracerUtility::mutateHeaders(request_headers, runtime);
-  }
+  mutateTracingRequestHeader(request_headers, runtime, config);
   mutateXfccRequestHeader(request_headers, connection, config);
 
   return final_remote_address;
 }
 
+void ConnectionManagerUtility::mutateTracingRequestHeader(Http::HeaderMap& request_headers,
```

Comment:
What was the purpose of moving this method out of the Tracing module?

Reply:
We need to pass config now, so it's a cyclic dependency if we pass it to the util.